packages/wallet/primitives/src/network.ts

```diff
@@ -1,630 +1,1229 @@
+    },
+    nativeToken: {
+      symbol: 'ETH',
+      name: 'Ether',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.TOY_TESTNET,
+    type: NetworkType.TESTNET,
+    name: 'toy-testnet',
+    title: 'TOY (Testnet)',
+    rpc: getRpcUrl('toy-testnet'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.TOY_TESTNET}.webp`,
+    blockExplorer: {
+      name: 'TOY Testnet Explorer',
+      rootUrl: 'https://toy-chain-testnet.explorer.caldera.xyz/',
+    },
+    nativeToken: {
+      symbol: 'TOY',
+      name: 'TOY',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.IMMUTABLE_ZKEVM,
+    type: NetworkType.MAINNET,
+    name: 'immutable-zkevm',
+    title: 'Immutable zkEVM',
+    rpc: getRpcUrl('immutable-zkevm'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.IMMUTABLE_ZKEVM}.webp`,
+    blockExplorer: {
+      name: 'Immutable zkEVM Explorer',
+      rootUrl: 'https://explorer.immutable.com/',
+    },
+    nativeToken: {
+      symbol: 'IMX',
+      name: 'IMX',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.IMMUTABLE_ZKEVM_TESTNET,
+    type: NetworkType.TESTNET,
+    name: 'immutable-zkevm-testnet',
+    title: 'Immutable zkEVM Testnet',
+    rpc: getRpcUrl('immutable-zkevm-testnet'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.IMMUTABLE_ZKEVM_TESTNET}.webp`,
+    blockExplorer: {
+      name: 'Immutable zkEVM Testnet Explorer',
+      rootUrl: 'https://explorer.testnet.immutable.com/',
+    },
+    nativeToken: {
+      symbol: 'IMX',
+      name: 'IMX',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.ROOT_NETWORK,
+    type: NetworkType.MAINNET,
+    name: 'rootnet',
+    title: 'The Root Network',
+    rpc: getRpcUrl('rootnet'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.ROOT_NETWORK}.webp`,
+    blockExplorer: {
+      name: 'The Root Network Explorer',
+      rootUrl: 'https://rootscan.io/',
+    },
+    nativeToken: {
+      symbol: 'XRP',
+      name: 'XRP',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.ROOT_NETWORK_PORCINI,
+    type: NetworkType.TESTNET,
+    name: 'rootnet-porcini',
+    title: 'The Root Network Porcini Testnet',
+    rpc: getRpcUrl('rootnet-porcini'),
+    logoURI: `https://assets.sequence.info/images/networks/medium/${ChainId.ROOT_NETWORK_PORCINI}.webp`,
+    blockExplorer: {
+      name: 'The Root Network Porcini Testnet Explorer',
+      rootUrl: 'https://porcini.rootscan.io/',
+    },
+    nativeToken: {
+      symbol: 'XRP',
+      name: 'XRP',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.HARDHAT,
+    type: NetworkType.LOCAL,
+    name: 'hardhat',
+    title: 'Hardhat (local testnet)',
+    rpc: 'http://localhost:8545',
+    nativeToken: {
+      symbol: 'ETH',
+      name: 'Ether',
+      decimals: 18,
+    },
+  },
+  {
+    chainId: ChainId.HARDHAT_2,
```

Comment:
We don't need to include these HARDHAT networks in the primitives, if required they can be defined within the test suite only.